### DISCUSSION TITLE
Welcome to json-schema-validator Discussions!

### DISCUSSION THREAD

**Discussion by stevehu** [networknt/json-schema-validator#740] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thulaban1990** [comment]
Hi, We use this package to validate JSON data with schema. We are storing JSON schemas in a AWS DynamodDB table as we have many of config like this. We have schemas created in a way that it promotes reusability where any schema could be re-used, its created as its own and stored in AWS DynamodDB and these are referenced $ref in other schemas where reusability is needed.

When getting schema to validate main schema, how to resolve and get schema from AWS DynamodDB while resolving $ref from main schema to get other reusable schema based on $ref.

If there is any example on how to fetch schema from custom data store helps us.
Looking forward to hear from you.

**Reply by stevehu** [reply]
The $ref works as a URL and it is better to create an API to retrieve the schema and output as a URL for the reference. This is how we use it in the light-portal to manage centralized schemas.
